### PR TITLE
xmlpull: Initial integration

### PR DESCRIPTION
**Pull request by onionpsy**
@oliverchang @kyakdan

### CODE REVIEW

File: projects/xmlpull/build.sh [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+#
+################################################################################
+
+mv $SRC/{*.zip,*.dict} $OUT
+
+cat > patch.diff <<- EOM
+diff --git a/pom.xml b/pom.xml
+index 3e29db9..c79e086 100644
+--- a/pom.xml	2022-04-14 17:14:14.830692400 +0200
++++ b/pom.xml	2022-04-14 17:33:25.535451222 +0200
```
Comment (oliverchang) [inline projects/xmlpull/build.sh:24]: This looks a bit brittle. What's the reason why we need to remove this?

**Review by oliverchang — COMMENTED** [review]
Also, can we add some dictionaries/seed corpora for this?

**Comment by oliverchang** [comment]
My bad, missed the dictionary/seed corpus here somehow. Merging!

File: projects/xmlpull/build.sh (hunk 1 above)
Comment (onionpsy) [reply projects/xmlpull/build.sh:24]: Agreed. I temporarily removed the javadoc because of dependency issue with maven. I'll look into fixing that.

**Pull request merged**